packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -266,13 +266,12 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
 
             address bondRecipient = resolvedPrice == numericalTrue ? assertion.asserter : assertion.disputer;
 
-            // If set to use UMA DVM as oracle then oracleFee must be sent to UMA Store contract. Else, if not using UMA
-            // DVM then the bond is returned to the correct party (asserter or disputer).
+            // Calculate oracle fee and the remaining amount of bonds to send to the correct party (asserter or disputer).
             uint256 oracleFee = (burnedBondPercentage * assertion.bond) / 1e18;
-            if (assertion.escalationManagerSettings.arbitrateViaEscalationManager) oracleFee = 0;
             uint256 bondRecipientAmount = assertion.bond * 2 - oracleFee;
 
-            // Send tokens. If the DVM is used as an oracle then send the oracleFee to the Store.
+            // Pay out the oracle fee and remaining bonds to the correct party. Note: the oracle fee is sent to the
+            // Store contract, even if the escalation manager is used to arbitrate disputes.
             if (oracleFee > 0) assertion.currency.safeTransfer(address(_getStore()), oracleFee);
```

Comment:
is there ever a case where `oracleFee==0`?

Reply:
only if for some reason bond was 0 (due to final fee being 0) or burned bond recipient was configured to 0.

Reply:
but agree this is not expected scenario and we could lift this check

Reply:
if it was to be 0 then this would just do a transfer event of 0 which seems fine.

Reply:
yes, and just realized burned bond percentage cannot be set to 0, so will lift it now